inc/command/class-command.php

```diff
@@ -212,19 +213,38 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 	 */
 	protected function codecept( InputInterface $input, OutputInterface $output ) {
 		$options = $input->getArgument( 'options' );
-		$module = $input->getOption( 'module' );
+		$tests_folder = $input->getOption( 'path' );
+		$output_folder = $input->getOption( 'output' );
 		$run_headless_browser = $input->getOption( 'browser' );
 		$use_chassis = $input->getOption( 'chassis' );
-		$tests_folder = $module !== 'project' ? "altis/$module/tests" : '../tests';
 		$project_subdomain = $this->get_project_subdomain();
 
+		$folders = [
+			'_data' => 'altis/dev-tools/tests/_data',
+			'_support' => 'altis/dev-tools/tests/_support',
+			'_envs' => 'altis/dev-tools/tests/_env',
+			'_output' => 'altis/dev-tools/tests/_output',
```

Comment:
This should default to `"{$tests_folder}/_output"`, I think codeception will create it if doesn't exist already.
```suggestion
			'_output' => "{$tests_folder}/_output",
```

Reply:
Yup, good catch!